packages/components/table/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@tradeshift/elements.table",
+  "version": "0.5.3",
+  "src": "src/table.js",
+  "main": "lib/table.umd.js",
+  "browser": "lib/table.umd.js",
+  "module": "lib/table.esm.js",
+  "files": [
+    "lib/*"
+  ],
+  "dependencies": {
+    "@tradeshift/elements": "^0.5.3",
+    "@tradeshift/elements.status": "^0.5.3",
+    "@tradeshift/elements.document-card": "^0.5.3"
```

Comment:
I believe these two dependencies can be removed.

Reply:
yes, forgot about them